### ISSUE TITLE
Standardize README shield badges

### ISSUE THREAD

**Issue by marcduiker** [dapr/rust-sdk#176]
We want to update the README for all Dapr repos so they all include shield badges for YouTube, Discord, X/Twitter and ensure they are all in the same style. See: https://github.com/dapr/community/issues/355

These are the [shield badges]( https://shields.io/badges) for the rust-sdk repo. 

[![Crates.io][crates-badge]][crates-url] [![Build Status][actions-badge]][actions-url] [![License: Apache 2.0][apache-badge]][apache-url] [![FOSSA Status][fossa-badge]][fossa-url] [![Good First Issues][gfi-badge]][gfi-url] [![discord][discord-badge]][discord-url] [![YouTube][youtube-badge]][youtube-link] [![X/Twitter][x-badge]][x-link]

[crates-badge]: https://img.shields.io/crates/v/dapr.svg?style=flat
[crates-url]: https://crates.io/crates/dapr
[apache-badge]: https://img.shields.io/github/license/dapr/rust-sdk?style=flat&label=License&logo=github
[apache-url]: https://github.com/dapr/rust-sdk/blob/master/LICENSE
[actions-badge]: https://github.com/dapr/rust-sdk/workflows/dapr-rust-sdk/badge.svg
[actions-url]: https://github.com/dapr/rust-sdk/actions?query=workflow%3Adapr-rust-sdk
[fossa-badge]: https://app.fossa.com/api/projects/custom%2B162%2Fgithub.com%2Fdapr%2Frust-sdk.svg?type=shield
[fossa-url]: https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2Fdapr%2Frust-sdk?ref=badge_shield
[gfi-badge]:https://img.shields.io/github/issues-search/dapr/rust-sdk?query=type%3Aissue%20is%3Aopen%20label%3A%22good%20first%20issue%22&label=Good%20first%20issues&style=flat&logo=github
[gfi-url]:https://github.com/dapr/rust-sdk/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
[discord-badge]: https://img.shields.io/discord/778680217417809931?label=Discord&style=flat&logo=discord
[discord-url]: http://bit.ly/dapr-discord
[youtube-badge]:https://img.shields.io/youtube/channel/views/UCtpSQ9BLB_3EXdWAUQYwnRA?style=flat&label=YouTube%20views&logo=youtube
[youtube-link]:https://youtube.com/@daprdev
[x-badge]:https://img.shields.io/twitter/follow/daprdev?logo=x&style=flat
[x-link]:https://twitter.com/daprdev

Snippet to include in the README:

```
[![Crates.io][crates-badge]][crates-url] [![Build Status][actions-badge]][actions-url] [![License: Apache 2.0][apache-badge]][apache-url] [![FOSSA Status][fossa-badge]][fossa-url] [![Good First Issues][gfi-badge]][gfi-url] [![discord][discord-badge]][discord-url] [![YouTube][youtube-badge]][youtube-link] [![X/Twitter][x-badge]][x-link]

[crates-badge]: https://img.shields.io/crates/v/dapr.svg?style=flat
[crates-url]: https://crates.io/crates/dapr
[apache-badge]: https://img.shields.io/github/license/dapr/rust-sdk?style=flat&label=License&logo=github
[apache-url]: https://github.com/dapr/rust-sdk/blob/master/LICENSE
[actions-badge]: https://github.com/dapr/rust-sdk/workflows/dapr-rust-sdk/badge.svg
[actions-url]: https://github.com/dapr/rust-sdk/actions?query=workflow%3Adapr-rust-sdk
[fossa-badge]: https://app.fossa.com/api/projects/custom%2B162%2Fgithub.com%2Fdapr%2Frust-sdk.svg?type=shield
[fossa-url]: https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2Fdapr%2Frust-sdk?ref=badge_shield
[gfi-badge]:https://img.shields.io/github/issues-search/dapr/rust-sdk?query=type%3Aissue%20is%3Aopen%20label%3A%22good%20first%20issue%22&label=Good%20first%20issues&style=flat&logo=github
[gfi-url]:https://github.com/dapr/rust-sdk/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
[discord-badge]: https://img.shields.io/discord/778680217417809931?label=Discord&style=flat&logo=discord
[discord-url]: http://bit.ly/dapr-discord
[youtube-badge]:https://img.shields.io/youtube/channel/views/UCtpSQ9BLB_3EXdWAUQYwnRA?style=flat&label=YouTube%20views&logo=youtube
[youtube-link]:https://youtube.com/@daprdev
[x-badge]:https://img.shields.io/twitter/follow/daprdev?logo=x&style=flat
[x-link]:https://twitter.com/daprdev
```

**Contributing**
- Comment `/assign` on this issue if you want to self-assign this issue and update the README.
- Don't forget to sign off your commit, for more details see [Developer Certificate of Origin (DCO) in the docs](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work).

@ maintainers: Please label this as `good first issue`.

**Comment by mikeee** [comment]
@MO2k4 Hi! Were there any blockers that we can assist with to get these PRs completed?

**Comment by MO2k4** [comment]
@mikeee no, no blockers I just took over everything and reduced it to just 3 😅
